README.md

```diff
@@ -10,27 +10,38 @@
   <h1 align="center">Apostrophe Code Upgrader: A2 ➡ A3</h1>
 </p>
 
-The Code Upgrader handles the majority of required modifications for an Apostrophe 2 (A2) codebase to run Apostrophe 3 (A3). It will also identify many specific lines and sections of code that a developer will need to convert manually.
+The Code Upgrader handles a portion of the required modifications for an Apostrophe 2 (A2) codebase to run Apostrophe 3 (A3). It will also identify many specific lines and sections of code that a developer will need to convert manually.
 
 **Status:** In development (not for production use)
 
 ## Purpose
 
 ### What it does
 
+#### The `lint` command
+
+The `lint` command scans your project for modifications that likely need to be made to be compatible with A3. The lint command will work well with basically all projects and detects many issues. Here are just a few examples of what the linter can detect:
+
+- The need to rename `lib/modules` to `modules`.
+- The need to change `{{ apos.area(...) }}` to `{% area ... %}`.
+- The need to move code from `construct()` to `methods()`, `handlers()`, etc.
+
+Since the linter is very tolerant it is a good candidate for use with nearly all A2 projects that are migrating to A3.
+
+#### The `upgrade` command
```

Comment:
Similarly, this material should be at the start of the "Running the code upgrade" section below.